src/styles/blocks/cards__container.scss

```diff
@@ -0,0 +1,29 @@
+.cards__container {
+  padding: 50px 40px;
+  display: grid;
+
+  grid-template-rows: 1fr 1fr;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  gap: 46px 48px;
+
+  justify-items: center;
+  justify-content: center;
+}
+
+@media (min-width: 488px) {
+  .cards__container{
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .cards__container{
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .cards__container{
+    --column-count: 4;
```

Comment:
Ensure there is an empty line before the `.cards__container` selector inside the media query to maintain consistency and follow the code style guidelines.